src/components/Link/Link.tsx

```diff
@@ -0,0 +1,20 @@
+import type { AnchorHTMLAttributes, PropsWithChildren } from "react";
+import type { Tone } from "../../tokens/colors";
+import type { FontSize, FontWeight } from "../../tokens/typography";
+
+interface LinkWithoutChildren extends AnchorHTMLAttributes<HTMLAnchorElement> {
```

Comment:
인터페이스 이름이 불필요하게 장황한 것 같습니다.

```suggestion
interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
```

Reply:
하단에서 PropsWithChildren로 감싸다보니 명확하게하려는 의도였는데 장황하다고 느낄수도있겠네요 수정하겠습니다.